src/namespaces.json

```diff
@@ -186,7 +186,9 @@
     "token": "0xC011a73ee8576Fb46F5E1c5751cA3B9Fe0af2a6F",
     "core": [],
     "min": 0,
-    "invalid": [],
+    "invalid": [
+      "QmZieYbP3Chs1T4hzmZerBXJ9t7P3bJDj3hJDoaPaGrqjL"
```

Comment:
@chefnomi It's the wrong space, sushi space is ☝️

Reply:
Oh sorry. Let me fix.